forPublication/vsm.md

```diff
@@ -48,39 +42,37 @@ Another client had a Product Owner signoff task which on average took several da
 wait time. In this case the development teams were very aware of the problem but the VSM exercise helped convince the 
 Product Owner through data that this quality gate was incredibly expensive. It became clear that the PO needed 
 to work more closely with the team throughout the iteration instead of validating features after implementation. 
-
 #### We are doing Scrum. I've been told a VSM does not apply for Scrum teams
 In Scrum, the three pillars Transparency, Inspection and Adaptation should enable autonomous, T-shaped teams to 
 self organise, optimize (and eliminate waste). In reality though, teams with various maturity levels struggle 
 to do so. Often they are not aware of the wasteful nature of certain activities and the hidden cost of handovers so they
 do not try to optimize for this.
-
 Scrum masters and Agile Coaches often fail to effectively facilitate and challenge teams in this area. 
 <br/><br/>
-I do make some adjustments for Scrum: The only official roles in a Scrum team are Dev, PO and SM I tend to focus on 
-the tasks and not work from roles. Furthermore (mature) Scrum teams tend to be less rigid and more fluid in terms of 
-the flow of work. Still, certain steps do always take place and lead to a picture as below, usually mirroring 
-the CI/CD pipeline with all the human tasks required for each step.
-
+I do make some adjustments for Scrum: The only official roles in a Scrum team are Dev, PO and SM.
+I tend to focus on the tasks and not work from roles.
+Furthermore (mature) Scrum teams tend to be less rigid and more fluid in terms of the flow of work.
+Still, certain steps do always take place and lead to a picture as below, usually mirroring the CI/CD pipeline with all the human tasks required for each step.
 #### Should I use specialized tooling?
 Tooling exists and can definitely make automation of the calculations easier, as well as repeatability. 
-I find the results are better using the walls though. The visual aspect of it is important 
-and through collaborating interactively important insights are shared and a common understanding is reached.
-
+I find the results are better using the walls though.
+The visual aspect of it is important and through collaborating interactively important insights are shared and a common understanding is reached.
 #### Other contexts where Value Stream Mapping can be provide Value
-Creating a DevOps, CI/CD pipeline. Finding bottlenecks and understanding the impact of (external) development dependencies.
+Creating a DevOps, CI/CD pipeline.
```

Comment:
I'm not sure I see the value of this section. I think if some activities in the pipeline are wasteful there is probably a process activity in the team that also shows this.